Packs/Listmonk/Integrations/Listmonk/Listmonk.py

```diff
@@ -0,0 +1,92 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import requests
+from typing import List, Dict, Any
+
+# Update the integration parameters to match your Listmonk setup
+LISTMONK_URL = demisto.params().get("listmonk_url", None)
+LISTMONK_USERNAME = demisto.params().get("listmonk_username", None)
+LISTMONK_PASSWORD = demisto.params().get("listmonk_password", None)
+
+
+class ListmonkClient:
+
+    def __init__(self, url: str, username: str, password: str):
+        self.base_url = url.rstrip('/')
+        self.auth = (username, password)
+
+    def get_lists(self) -> Dict[str, Any]:
+        endpoint = f"{self.base_url}/api/lists"
+        response = self._send_request("GET", endpoint)
+        lists = response.json()
+        lists_entry = {
+            "Type": entryTypes['note'],
+            "Contents": lists,
+            "ContentsFormat": formats['json'],
+            "ReadableContentsFormat": formats['markdown'],
+            "HumanReadable": tableToMarkdown('Listmonk Lists', lists),
+            "EntryContext": {"Listmonk.ListsResults": createContext(lists)}
+        }
```

Comment:
Another option, is to use CommandResult. you can view implemantion example in HelloWorld pack.

for more information: https://xsoar.pan.dev/docs/integrations/context-and-outputs#in-the-code